src/server/frontend_wayland/window_wl_surface_role.cpp

```diff
@@ -40,6 +40,13 @@ namespace ms = mir::scene;
 namespace msh = mir::shell;
 namespace geom = mir::geometry;
 
+namespace
+{
+static geom::Size const max_possible_size{
```

Comment:
You need the `static` here; anonymous namespace implicitly specifies static duration and internal linkage.

That said, it's entirely harmless.